lib/backburner.js

```diff
@@ -26,6 +26,11 @@ export default function Backburner(queueNames, options) {
   this._debouncees = [];
   this._throttlers = [];
   this._timers = [];
+  this._eventNames = ['end', 'begin'];
+  this._eventCallbacks = {};
+  for (var i = 0; i < this._eventNames.length; i++) {
+    this._eventCallbacks[this._eventNames[i]] = [];
```

Comment:
Wanted to make this general in case we wanted more events in the future, but if you don't think there are going to be more, I can certainly take this out.